.github/workflows/test_with_woodwork_main_branch.yml

```diff
@@ -0,0 +1,59 @@
+name: Test Featuretools with Woodwork main Branch
+on:
+  schedule:
+    - cron: '0 * * * *'
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
```

Comment:
let's only do schedule and workflow dispatch for now

Reply:
updated